src/tests/test_alldebrid_downloader.py

```
@@ -46,91 +51,51 @@ def get(url, **params):
-    assert torrent_id == 251993753
 
+def test_select_files(downloader):
+    assert downloader.select_files(MAGNET_ID, [1, 2, 3]) == None
 
-def test_add_torrent_cached(upload_ready):
-    alldebrid.get = upload_ready
-    torrent_id = add_torrent(UBUNTU)
-    assert torrent_id == 251993753
 
+def test_get_torrent_info(downloader):
+    torrent_info = downloader.get_torrent_info(MAGNET_ID)
```

Ruff (0.8.0)

Local variable `torrent_info` is assigned to but never used

Remove assignment to unused variable `torrent_info`

(F841)